sale_blanket_order/models/blanket_orders.py

```diff
@@ -98,7 +98,7 @@ def _compute_amount_all(self):
             ("draft", "Draft"),
             ("open", "Open"),
             ("done", "Done"),
-            ("expired", "Expired"),
```

Comment:
I updated the module with both states. If you see anything else that smells please let me know

Reply:
> I understand the idea behind this change BUT:
> 
> I think, from a functional point of view, a blanket order has an expiration date because of its nature (of blanket order) and should have that 'expired' state. Because of that, blanket order should differ from sale order.
> 
> For the remaining changes, I'm ok

The main idea driving this change is to make the module compatible with base_revision, which expects a state `cancel`. This is why the first draft of this PR changed `expire` to `cancel`.

What do you think about having the state be called `cancel` in the backend for compatibility, and "Cancelled/Expired" in the frontend to cover both use cases?